packages/storage-macro/src/lib.rs

```diff
@@ -5,6 +5,25 @@ use syn::{
     parse_macro_input, ItemStruct,
 };
 
+/// Auto generate an `IndexList` impl for your indexes struct.
+///
+/// # Example
+///
+/// ```rust
+/// #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+/// struct TestStruct {
+///     id: u64,
+///     id2: u32,
+///     addr: Addr,
+/// }
+///
+/// #[index_list(TestStruct)] // <- Add this line right here.
+/// struct TestIndexes<'a> {
+///     id: MultiIndex<'a, u32, TestStruct, u64>,
+///     addr: UniqueIndex<'a, Addr, TestStruct>,
+/// }
+/// ```
+///
```

Comment:
Ah, so normally macro docs would live in the `storage-plus` crate, above the reexport. [Here's an example.](https://github.com/CosmWasm/cosmwasm/blob/main/packages/schema/src/lib.rs#L12-L36)

Does this example pass when tested?

Reply:
Good point, it does not. I'll move it to the reexport